### ISSUE TITLE
Missing path

### ISSUE THREAD

**Issue by Hideman85** [40thieves/webpack-sentry-plugin#90]
`WebpackSentryPlugin: unable to delete 'static/2.1.13/js/59.afdcc3b4.chunk.js.map'. File does not exist; it may not have been created due to a build error.`

webpack config.output.path = 'build'

So the file is located at `build/static/2.1.13/js/59.afdcc3b4.chunk.js.map`

**Comment by kambala3000** [comment]
@Hideman85 hi! Any solutions?

**Comment by Hideman85** [comment]
Yes leave delete option to false and add a "post-build" entry in script section of your package.json  and use rimraf to delete files by yourself.
